public/Query.js

```diff
@@ -769,4 +771,31 @@ Query.executeFromSearch = function () {
 
 window.addEventListener('popstate', function () {
 	Query.executeFromSearch()
-})
\ No newline at end of file
+})
+
+/* Make table header fixed after scroll limit is reached */
+function fixedHeader() {
+
+	document.getElementById('fixed-table-header').innerHTML = ''
```

Comment:
Use [`Panel.get`](https://github.com/clubedaentrega/node-mongo-admin/blob/master/public/Panel.js#L11) instead of `document.getElementById`